scripts/lib/aws/elasticache.sh

```diff
@@ -12,6 +12,29 @@ SCRIPTS_DIR="$ROOT_DIR/scripts"
 # functions for tests
 #################################################
 
+# k8s_controller_apply_temp_creds adds AWS temporary credentials
+# to service controller running on kubernetes cluster.
+# it depends upon:
+#   $service_name
+#   $AWS_ACCESS_KEY_ID
+#   $AWS_SECRET_ACCESS_KEY
+#   $AWS_SESSION_TOKEN
+k8s_controller_apply_temp_creds() {
+  kubectl -n ack-system set env deployment/ack-"$service_name"-controller \
+    AWS_ACCESS_KEY_ID="$AWS_ACCESS_KEY_ID" \
+    AWS_SECRET_ACCESS_KEY="$AWS_SECRET_ACCESS_KEY" \
+    AWS_SESSION_TOKEN="$AWS_SESSION_TOKEN" 1>/dev/null
+}
+
+# reload_temp_credentials generates AWS temporary credentials
+# and adds them to service controller running on kubernetes cluster.
+reload_temp_credentials() {
```

Comment:
Please merge this function with the `k8s_controller_apply_temp_creds` function above and name it `k8s_controller_reload_credentials`. Please also place this into the `scripts/lib/k8s.sh` file so it can be reused across different services' tests

Reply:
Updated scripts.